docs/yaml.rst

```diff
@@ -457,10 +457,139 @@ syear     Year from a given date.
+      `Blogin` HPCs, which are included in `FESOM`'s configuration file
+      (``esm_tools/configs/components/fesom/fesom.yaml``):
+
+      .. code-block:: yaml
+
+         [ ... ]
+
+         compiletime_environment_changes:
+                 add_export_vars:
+                         takenfrom:      fesom1
+         choose_computer.name:
+                 mistral:
+                         add_compiletime_environment_changes:
+                                 add_module_actions:
+                                         - "unload gcc"
+                                         - "load gcc/4.8.2"
+                 blogin:
+                         add_compiletime_environment_changes:
+                                 add_export_vars:
+                                         - "NETCDF_DIR=/sw/dataformats/netcdf/intel.18/4.7.3/skl/"
+                                         - "LD_LIBRARY_PATH=$NETCDF_DIR/lib/:$LD_LIBRARY_PATH"
+                                         - "NETCDF_CXX_INCLUDE_DIRECTORIES=$NETCDF_DIR/include"
+                                         - "NETCDF_CXX_LIBRARIES=$NETCDF_DIR/lib"
+                                         - "takenfrom='fesom1'"
+
+         runtime_environment_changes:
+                 add_export_vars:
+                         AWI_FESOM_YAML:
+                                 output_schedules:
+                                         -
+                                                 vars: [restart]
+                                                 unit: ${restart_unit}
+                                                 first: ${restart_first}
+                                                 rate: ${restart_rate}
+                                         -
+                                                 [ ... ]
+
+      Independently of the computer, ``fesom.yaml`` exports always the ``takenfrom``
+      variable for compiling. Because ``compiletime_environment_changes`` is already
+      defined for that purpose, any ``compiletime_environment_changes`` in a
+      ``choose_`` block needs to have an ``add_`` at the beginning. Here we see that a
+      ``choose_`` block is used to select which changes to apply compile environment
+      (``add_compiletime_environment_changes``) depending on the HPC system we are in
+      (`Mistral` or `Blogin`). For more details on how to use the ``choose_`` and
+      ``add_`` functionalities see :ref:`yaml:Switches (\`\`choose_\`\`)` and
+      :ref:`yaml:Append to an Existing List (\`\`add_\`\`)`.
+
+      We also see here how ``runtime_environment_changes`` is used to add nested
+      information about the output schedules for `FESOM` into an
+      ``AWI_FESOM_YAML`` variable that will be exported to the runtime environment.
+
 Changing Namelists
-~~~~~~~~~~~~~~~~~~
+------------------
 
-It is also possible to specify namelist changes to a particular section:
+It is also possible to specify namelist changes to a particular section of a namelist:
```

Comment:
Below the example you state that the namelist variable will be set to `.false.` but in the yaml file you say `false`. It occures probably somewhere else, but is it needed to state it in the yaml file without points? 
Are such boolean variables interpreted by python? I remember having problems with it, that these variables end up being upper case and I had to lowercase them all for a xml configuration file.

Reply:
Thanks for pointing out this :) , I added a paragraph after the example.